clients/ui/frontend/src/app/services/modelRegistrySettingsService.ts

```diff
@@ -0,0 +1,97 @@
+import * as _ from 'lodash-es';
```

Comment:
@lucferbux I am not sure whether we need this file because we have these endpoints in k8s.ts file